packages/wdio-reporter/src/index.ts

```diff
@@ -40,6 +41,7 @@ export default class WDIOReporter extends EventEmitter {
     }
     retries = 0
     runnerStat?: RunnerStats
+    contentPresent: boolean
```

Comment:
How about renaming this flag to `isContentPresent` ?

Reply:
Done in the latest commit.